app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -64,6 +64,13 @@ def path_to_regex
             "/#{path}/.*|/#{path}/?"
           end
         end
+
+        def replace_path
+          return {} if path.to_s.empty?
```

Comment:
Is it a case where we need to cast it with `to_s`?

```suggestion
          return {} if path.blank?
```